tests/unit-tests/VolleyM.Domain.Framework.UnitTests/Authorization/AuthorizeUserSteps.cs

```diff
@@ -5,25 +5,26 @@
 using System.Security.Claims;
 using System.Security.Principal;
 using System.Threading.Tasks;
+using TechTalk.SpecFlow;
 using VolleyM.Domain.Contracts;
 using VolleyM.Domain.Contracts.Crosscutting;
 using VolleyM.Domain.Framework.Authorization;
 using VolleyM.Domain.IdentityAndAccess;
 using VolleyM.Domain.IdentityAndAccess.Handlers;
 using VolleyM.Domain.IdentityAndAccess.RolesAggregate;
 using VolleyM.Domain.UnitTests.Framework;
-using Xunit.Gherkin.Quick;
 
 namespace VolleyM.Domain.Framework.UnitTests.Authorization
 {
-    [FeatureFile(@"./Authorization/AuthorizeUser.feature")]
+    [Binding]
+    [Scope(Feature = "Authorize User")]
```

Comment:
How-To: Example 12: Scope attribute. Should match Feature name in the feature file